modules/native_streaming_client_module/src/native_device_impl.cpp

```diff
@@ -405,6 +405,11 @@ void NativeDeviceImpl::setConnectionString(const StringPtr& connectionString)
         }
     }
 
+    if (!newDeviceInfo.hasProperty("NativeConfigProtocolVersion"))
+    {
+        newDeviceInfo.addProperty(IntProperty("NativeConfigProtocolVersion", clientComm->getProtocolVersion()));
+    }
```

Comment:
Should this be a string property?

Reply:
I want to compare the versions in an orderly. Therefore I used integer.